Jenkinsfile

```diff
@@ -44,7 +44,7 @@ pipeline {
 			}
 			post {
 				always {
-					archiveArtifacts artifacts: '*.log,*/target/work/data/.metadata/*.log,*/tests/target/work/data/.metadata/*.log,apiAnalyzer-workspace/.metadata/*.log', allowEmptyArchive: true
+					archiveArtifacts artifacts: '*.log,*/target/work/data/.metadata/*.log,*/tests/target/work/data/.metadata/*.log,apiAnalyzer-workspace/.metadata/*.log,repository/target/repository/**', allowEmptyArchive: true
```

Comment:
As you're not enabling the profile here this would be empty thus not needed. Have you forgotten to actually enable the maven profile?

Reply:
The current state indeed mixes things. Should we enable the new profile by default in CI (I'm in favor of it)?

Reply:
Yes, please. Once the verification build is complete, if it slowsdown significanlty all changes to jenkinsfile will have to be reverted.

Reply:
$ time mvn clean verify -DskipTests -Pp2-repo
real	1m23.959s
user	3m36.804s
sys	0m8.281s

$ time mvn clean verify -DskipTests
real	1m27.132s
user	4m47.082s
sys	0m9.953s

So difference is neglectable (actually it's totally invisible under any network latency)
The publishArtifacts step might take a couple more seconds to complete though.

So I agree that it's worth making it default as the price seems neglectable.